app/utils/theme/index.ts

```diff
@@ -62,7 +62,7 @@ export function getComponents(inColor: string): {red: number; green: number; blu
     };
 }
 
-export function makeStyleSheetFromTheme(getStyleFromTheme: (a: any) => any): (a: any) => any {
+export function makeStyleSheetFromTheme(getStyleFromTheme: (a: any) => any): (a?: any) => any {
```

Comment:
not comfortable with this change

Reply:
If we don't then this line complains.  https://github.com/mattermost/mattermost-mobile/blob/dbd80f22cd3419657765fd060343904cf0173697/app/screens/forgot_password/index.tsx#L24

Reply:
Yes please :)